cpp/data/sql_updates/ub_tools.19

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE metadata_presence_tracer MODIFY COLUMN metadata_field_name CHAR(3) NOT NULL;
```

Comment:
i guess this file should be renamed to `ub_tools.22` (multiple updates in the meantime)